Modulefile

```diff
@@ -1,5 +1,5 @@
 name    'ghoneycutt-dnsclient'
-version '3.1.1'
+version '3.2.0'
```

Comment:
Do not change the version when sending PR's as the maintain will do that when they cut a release. For instance, I might merge another PR before this one and then this version would not be correct.